nerfstudio/process_data/equirect_utils.py

```diff
@@ -22,18 +22,139 @@
 import cv2
 import numpy as np
 import torch
-from equilib import Equi2Pers
-from rich.progress import (
-    BarColumn,
-    Progress,
-    TaskProgressColumn,
-    TextColumn,
-    TimeRemainingColumn,
-)
+from rich.progress import BarColumn, Progress, TaskProgressColumn, TextColumn, TimeRemainingColumn
 
 from nerfstudio.utils.rich_utils import CONSOLE, ItersPerSecColumn
 
 
+# https://gist.github.com/fgolemo/94b5caf0e209a6e71ab0ce2d75ad3ed8
+def rotation_matrix(axis: torch.Tensor, theta: torch.float) -> torch.Tensor:
```

Comment:
I think this functionality already exists - https://github.com/nerfstudio-project/nerfstudio/blob/53c7c8eb07700ab087d36d18665175b39ab093c2/nerfstudio/cameras/camera_utils.py#L451

Reply:
oh awesome, I'll use that

Reply:
so it turns out the functions were different enough so I just kept the new one but renamed it to decrease confusion. The new one implements Euler-Rodriquez rotation.